easybuild/easyconfigs/f/FLAC/FLAC-1.3.3-GCCcore-10.2.0.eb

```
@@ -0,0 +1,28 @@
+easyblock = 'ConfigureMake'
+
+name = 'FLAC'
+version = '1.3.3'
+
+homepage = 'https://xiph.org/flac/'
+description = """FLAC stands for Free Lossless Audio Codec, an audio format similar to MP3, but lossless, meaning
+that audio is compressed in FLAC without any loss in quality."""
+toolchain = {'name': 'GCCcore', 'version': '10.2.0'}
+
+source_urls = ['https://ftp.osuosl.org/pub/xiph/releases/flac/']
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['213e82bd716c9de6db2f98bcadbc4c24c7e2efe8c75939a1a84e28539c4e1748']
+
+builddependencies = [('binutils', '2.35')]
+
+dependencies = [('libogg', '1.3.4')]
+
+configopts = '--enable-static --enable-shared'
+
+sanity_check_paths = {
+    'files': ['lib/libFLAC.a', 'lib/libFLAC++.a',
+              'lib/libFLAC.%s' % SHLIB_EXT, 'lib/libFLAC++.%s' % SHLIB_EXT,
+              'bin/flac'],
+    'dirs': ['include/FLAC', 'include/FLAC++'],
+}
```

(I checked, this works)

```suggestion
sanity_check_paths = {
    'files': ['bin/flac', 'lib/libFLAC.a', 'lib/libFLAC++.a',
              'lib/libFLAC.%s' % SHLIB_EXT, 'lib/libFLAC++.%s' % SHLIB_EXT],
    'dirs': ['include/FLAC', 'include/FLAC++'],
}

sanity_check_commands = ["flac --help"]
```